api/relations/v1/relation_tuples.proto

```diff
@@ -0,0 +1,60 @@
+syntax = "proto3";
+
+package api.relations.v1;
+
+import "google/api/annotations.proto";
+import "relations/v1/common.proto";
+
+option go_package = "ciam-rebac/api/relations/v1;v1";
+option java_multiple_files = true;
+option java_package = "api.relations.v1";
+
+service RelationTupleService {
```

Comment:
What are we going to do about `WriteTuples`? Future PR? Without we can't do a transactional create and delete.

There's possibly an argument for having both "write" and "create/delete". From a UX POV, I'd probably use the more specific thing (e.g. delete) if I could, rather than having to worry about the more complicated semantics of a write call.

Reply:
For the record I _think_ we settled on leaving existing operations for convenience, and adding the more capable `Write` RPC later (keep me honest)